float32 factor
arm_operation/joint_value[] joints
---
string plan_result
